PictureApp.API/Helpers/AutoMapperProfiles.cs

```diff
@@ -23,15 +23,18 @@ private void FromDomainToDto()
             CreateMap<User, UserForRegisterDto>()
                 .ForMember(dest => dest.Password, opt => opt.Ignore());
             CreateMap<User, UserLoggedInDto>();
+            CreateMap<Photo, PhotosForPhotoExploreViewDto>();
             CreateMap<User, UserForDetailedDto>()
                 .ForMember(dest => dest.PhotoUrl,
-                    opt => { opt.MapFrom(src => src.Photos.FirstOrDefault(p => p.IsMain).Url); }).ForMember(
-                    dest => dest.ActivationToken,
+                    opt => { opt.MapFrom(src => src.Photos.FirstOrDefault(p => p.IsMain).Url); })
```

Comment:
In case of null from src.Photos.FirstOrDefault() we will receive null exception. I would rather encourage You to use something like this: 
```
src => { 
var url = src.Photos.FirstOrDefault();
return url == null ? string.Empty : url;
}
```
instead.